0009-salt-working-groups.md

```diff
@@ -0,0 +1,121 @@
+
+## Timelines 
+
+Based on the applications submitted, salt chooses the final working group members 
+
+Below is the approximate guidelines for how long each working group term last. The actual timelines might differ. For e.g. Testing and Release Working Group might decide to convene immediately after a major release and dissolve after the next major release. 
+
+![Working Group Timelines](./diagrams/Working-group-timelines.svg)
+
+
+
+*T0 is feature freeze date of current major release
+ T1 is feature freeze date of next major release
+
+
+Working Groups will meet once a month for 60 mins. Below are some guidelines to working group participants although each of these groups may operate a little differently depending on their needs and workflow.
+
+## Working Group Structure  
+
+Each working group will have folks participating in 3 types of roles: Captain, Participant and Facilitator. 
+
+### Captain
+
+Each working group’s captain will help lead their respective working group. 
+
+#### Role of Captains
+- The captain helps shepherd a subset of features/critical fixes for a given release
+- Captain will help coordinate other contributors' work in their area
+- Captains will attend monthly working group meetings for their respective working group. SaltStack will set these up, but SaltStack will hold no restriction on a Captain's wish to hold any additional meetings with their respective working group
+- Single review completed by the captain on the PR will be considered as criteria met to merge the PR (No additional reviews required) 
+
+#### Captain Guidelines
+
+- Captain will have merge rights to Salt once a PR has been reviewed and approved
+- A captain of one working group cannot be captain of any other salt working groups. But captain can be a participant of other working groups 
+- Captain can vote to choose the captain and/or participants of next term working group
+- Captains have a 3 successive term limit. Captains can take a gap after their 3 successive terms and come back to be a captain after a gap term
+- Starting from 2nd term working groups, the captains chosen need to have been a participant in at least one working group 
+- Once a captain’s term end and they are no longer the captain of the next term, their merge rights will be rescinded and transferred to new/incoming captain of the working group 
+- Captain’s need to follow the [Merging Guidelines](https://github.com/saltstack/salt-enhancement-proposals/); This is to ensure Salt project grows responsibly. 
+- In rare occasions and in consultation with Salt Core team, Captain can nominate a Vice Captain for the working group, to help assist during long leave of absence or to share responsibilities. Captain can also give merge access to said Vice Captain of their working group while ensuring they follow the Merging Guidelines.
```

Comment:
- In rare occasions and in consultation with `the` Salt `core` team, `the captain` can nominate a `vice-captain` for the working group, to help assist during `a` leave of absence or to share responsibilities. `The captain` can also give merge access to the `vice-captain`. The `vice-captain` must adhere to the same guidelines as the captain.